docs/source/app_developers_guide/ubuntu.rst

```diff
@@ -2,488 +2,716 @@
-and component communications. To change the interface and port used, the
-`--bind` flag can be used. See :doc:`/cli/sawtooth-validator` for more
-information on the available flags.  The following command is equivalent to the
-default behavior::
+* In the ``sawtooth-validator`` command above, the ``-vv`` option sets the log
+  level to include DEBUG messages. To run the validator with less logging
+  output, use ``-v`` or omit the option.
 
-    sudo -u sawtooth sawtooth-validator -vv --bind network:tcp://127.0.0.1:8800 --bind component:tcp://127.0.0.1:4004
+* By default, the validator listens on the loopback interface for both network
+  and component communications. To change the interface and port, use the
+  ``--bind`` option. The following command is equivalent to the default
+  behavior:
 
-The validator can process transactions in serial (the default) or parallel
-with no difference in the state produced. To process in parallel, use the
-option ``--scheduler parallel``. To get the most benefit from the parallel
-option, start multiple transaction processors for the types of transactions
-where there is an expected high volume.
+  .. code-block:: console
 
+     sudo -u sawtooth sawtooth-validator -vv --bind network:tcp://127.0.0.1:8800 --bind component:tcp://127.0.0.1:4004
 
-Starting the REST API
-=====================
+* The validator can process transactions in serial or parallel. By default, the
+  validator uses serial scheduling. Parallel scheduling provides the most
+  benefit when many transaction processors are running on the validator node.
 
-In order to configure a running validator, submit batches, and query the state
-of the ledger, you must start the REST API application.
+  To process transactions in parallel, use the option ``--scheduler parallel``.
 
-Open a new terminal window, then run the following command to start the
-REST API and connect to a local validator:
+For more information on validator options, see
+:doc:`../cli/sawtooth-validator` in the CLI Command Reference.
 
-.. code-block:: console
 
-  $ sudo -u sawtooth sawtooth-rest-api -v
+Step 4: Start the REST API
+==========================
 
-Starting and Configuring the Transaction Processors
-===================================================
+The REST API allows you to configure a running validator, submit batches, and
+query the state of the distributed ledger.
 
-This section describes how to start the IntegerKey and Settings family
-transaction processors, confirm that the REST API is running, and tell
-the validator or validator network to accept transactions from these
-transaction families.
+#. Open a new terminal window (the rest-api terminal window). In this procedure,
+   the prompt ``user@rest-api$`` shows the commands that should be run in this
+   window.
 
-Transaction processors can be started either before or after the validator is
-started.
+#. Run the following command to start the REST API and connect to the local
+   validator.
```

Comment:
I'll add a link for the REST API and `settings-tp`, but I can't do this for `intkey-tp-python` and `xo-tp-python` because they aren't in the CLI Command reference. Does anyone know why they're not there?

As a work-around (for now), I can say "Run ``intkey-tp-python --help`` and "Run ``xo-tp-python --help``".